contracts/modules/capital/Ramm.sol

```diff
@@ -410,9 +409,9 @@ contract Ramm is IRamm, MasterAwareV2, ReentrancyGuard {
       //   ? set n(new) = n(BV)
       //   : set n(new) = n(R)
 
-      return bufferedCapital * nxm + bufferedCapital * nxm * r / RATCHET_PERIOD / RATCHET_DENOMINATOR > eth * context.supply
+      return bufferedCapital * nxm + bufferedCapital * nxm * elapsed * NORMAL_RATCHET_SPEED / RATCHET_PERIOD / RATCHET_DENOMINATOR > eth * context.supply
```

Comment:
the comment above is now out-of-date, let's update it / make it more explicit

Reply:
it's not really out of date, it more that it represents the formula we had on paper